andela/activo-web
# #158563892 Make field rows when adding categories draggable

**What does this PR do?**

This PR allows users to  drag and drop field rows when creating a category 

**Description of Task to be completed?**

When a user clicks and holds the left edge of the field they are able to drag and drop field to rearrange the order of attributes as it will display in the add asset form.

**How should this be manually tested?**

After cloning and running app, go to the Settings page and click on one of the create new category option

- Hover your pointer over the left edge of the attributes. The pointer icon should change to a move icon

- Drag and drop the attribute up or down. The attribute position should change according to your drag motion.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#158563892 User can drag and drop field rows when creating a category](https://www.pivotaltracker.com/story/show/158563892)

**Screenshots (if appropriate)**

**When dragging**

<img width="1440" alt="screen shot 2018-07-30 at 16 31 31" src="https://user-images.githubusercontent.com/17454824/43400315-14cf3f02-9416-11e8-8180-a8ec5defe874.png">

**After dragging**

<img width="1440" alt="screen shot 2018-07-30 at 16 30 19" src="https://user-images.githubusercontent.com/17454824/43400338-266e903c-9416-11e8-990d-1ce573d4b323.png">

**Questions:**

N/A
